packages/@react-spectrum/s2/src/NotificationBadge.tsx

```diff
@@ -40,7 +40,8 @@ export interface NotificationBadgeProps extends DOMProps, AriaLabelingProps, Sty
 }
 
 interface NotificationBadgeContextProps extends Partial<NotificationBadgeProps> {
-  isDisabled?: boolean
+  isDisabled?: boolean,
+  staticColor?: 'black' | 'white' | 'auto'
```

Comment:
Since in practice the NotificationBadge either has `color: auto` applied or not, I wonder whether the prop set here should accept `black`/`white` or if it should just be a boolean. Maybe it could always be auto (but would need to assume it would always get its color from a parent)?